doc/index.rst

```diff
@@ -78,83 +109,83 @@ The Sprint in which these tasks are placed is defined :ref:`here <procedures/jir
-    
-  * Write the code.
-  * Write the unit tests.
-  * Update documentation
-  * When complete, move to In Review.
+* Create the ticket branch in the git repo and link the branch to the JIRA component.
 
-    * Complete meaning:
+  * No active development is ever done on the Master or Develop branches.
+  
+* Write the code.
+* Write the unit tests.
+* Update documentation
+* When complete, move to In Review.
 
-      * Add a link of the PR to the ticket.
-      * Unit tests exist, have been successfully run and results have been added to the ticket or PR.
-      * Add a link to the generated documentation site
+  * Complete meaning:
 
-* In Review
+    * Add a link of the PR to the ticket.
+    * Unit tests exist, have been successfully run and results have been added to the ticket or PR.
+    * Add a link to the generated documentation site
 
-  * Once the code is complete and all unit tests are passing, initiate a pull request on the develop branch and assign it to the Reviewer(s).
-  * The Reviewer ensures
+**In Review**
 
-    * Code is complete and understandable.
-    * CI is passing.
-    * Documentation is done, including a reference to the lsst.io site
-    * All requirements, as defined in the ticket description, are met.
+* Once the code is complete and all unit tests are passing, initiate a pull request on the develop branch and assign it to the Reviewer(s).
+* The Reviewer ensures
 
-  * If issues are found
+  * Code is complete and understandable.
+  * CI is passing.
+  * Documentation is done, including a reference to the lsst.io site
+  * All requirements, as defined in the ticket description, are met.
 
-    * Update Jira with issue
+* If issues are found
 
-      * Comments on GitHub PR are considered acceptable as well
- 
-    * Sends back to developer
+  * Update Jira with issue
 
-  * If no issues are found
+    * Comments on GitHub PR are considered acceptable as well
 
-    * Moves Jira ticket to review complete with approval/minor changes.
+  * Sends back to developer
 
-* Reviewed
+* If no issues are found
 
-  * The Developer then merges the pull-request (see Merge Process, below).
-  * Moves ticket to Resolved.
+  * Moves Jira ticket to review complete with approval/minor changes.
 
-* Done
+**Reviewed**
 
-  * This is the purview of QA.
-  
-    * QA does another spot review, to ensure the requirements are spelled out and properly met, all documentation is provided and the unit tests are passing.
-    * This is also when QA works on the higher level tests (Functional, Integration, etc).
-    * Once this is Complete, QA moves to Closed.
+* The Developer then merges the pull-request.
+* Moves ticket to Resolved.
 
-  * The ticket is complete.
+**Done**
 
-    * Feature was successfully implemented.
-    * Feature was invalid; proper explanation provided.
-    * Ticket was already fixed; proper explanation provided.
+* This is the purview of QA.
```

Comment:
I don't think we are doing or intend on doing any the following. I'd remove it altogether